src/components/calcite-pick-list/calcite-pick-list.scss

```diff
@@ -1,12 +1,14 @@
 :host {
-  @extend %component-host;
   @apply items-stretch
     bg-transparent
     flex
     flex-col
     relative
     flex-grow
-    flex-shrink-0;
+    flex-shrink-0
+    box-border
+    text-color-2
+    text--1h;
```

Comment:
This is missing the wildcard selector and box-border style from the component-host partial.

Reply:
@justinhough, this one too (add the wild card selector and box-border)